spacelift/resource_drift_detection.go

```diff
@@ -79,9 +85,10 @@ func resourceDriftDetectionCreate(ctx context.Context, d *schema.ResourceData, m
 	variables := map[string]interface{}{
 		"stack": toID(stackID),
 		"input": structs.DriftDetectionIntegrationInput{
-			Reconcile: graphql.Boolean(d.Get("reconcile").(bool)),
-			Schedule:  scheduleExpressions,
-			Timezone:  graphql.NewString(graphql.String(d.Get("timezone").(string))),
+			Reconcile:   graphql.Boolean(d.Get("reconcile").(bool)),
+			IgnoreState: graphql.Boolean(d.Get("ignore_state").(bool)),
```

Comment:
what happens if it's nil? defaults to false?

Reply:
Default value in DB is `false`, all entries have a value set in the backend, if not provided default is used.